packages/atomic/src/pages/examples/commerce-website/listing-surf-accessories.html

```diff
@@ -25,7 +26,88 @@ <h1>Product listing page</h1>
     <main>
       <section>
         <atomic-commerce-interface type="product-listing">
-          <atomic-commerce-product-list> </atomic-commerce-product-list>
+          <atomic-commerce-product-list display="grid" density="compact" image-size="small">
+            <atomic-product-template>
+              <template>
+                <style>
+                  .field {
+                    display: inline-flex;
+                    align-items: center;
+                  }
+
+                  .field-label {
+                    font-weight: bold;
+                    margin-right: 0.25rem;
+                  }
+
+                  .thumbnail {
+                    display: none;
+                    width: 100%;
+                    height: 100%;
+                  }
+
+                  .icon {
+                    display: none;
+                  }
+
+                  .brand {
+                    font-weight: 600;
+                  }
+
+                  .result-root.image-small .thumbnail,
+                  .result-root.image-large .thumbnail {
+                    display: inline-block;
+                  }
+
+                  .result-root.image-icon .icon {
+                    display: inline-block;
+                  }
+
+                  .result-root.image-small atomic-result-section-visual,
+                  .result-root.image-large atomic-result-section-visual {
+                    border-radius: var(--atomic-border-radius-xl);
+                  }
+
+                  .salesforce-badge::part(result-badge-element) {
+                    background-color: #44a1da;
+                    color: white;
+                  }
+                </style>
+                <atomic-result-section-visual>
```

Comment:
We should not have any `result-section` components in commerce: We'll need to spin up new ones specific for commerce.

It's nice because it looks better now, but I'd prefer if we kept those sample page as pure as possible, as it guides us to identify what still needs to be done when things are "ugly" ;)